lib/services/auth/service.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:liga_inggris_mobile/app/core/base/base_service.dart';
+import 'package:liga_inggris_mobile/app/core/handler/service_result.dart';
+import 'package:liga_inggris_mobile/services/auth/model.dart';
+import 'package:liga_inggris_mobile/services/club/model.dart';
+import 'package:shared_preferences/shared_preferences.dart';
```

Comment:
![image](https://github.com/user-attachments/assets/4219217f-1705-46a8-8e9a-93e0da685e7c)

```suggestion
import 'package:flutter/material.dart';
import 'package:get/get.dart';
import 'package:liga_inggris_mobile/app/core/base/base_service.dart';
import 'package:liga_inggris_mobile/services/auth/model.dart';
import 'package:shared_preferences/shared_preferences.dart';
```